src/main/java/com/example/controllers/AuthenticationController.java

```diff
@@ -0,0 +1,45 @@
+package com.example.controllers;
+
+import com.example.dto.AuthTokenDto;
+import com.example.dto.UserDtoForLogin;
+import com.example.filters.TokenProvider;
+import com.example.services.UserService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.annotation.Resource;
+
```

Comment:
JD?

Reply:
@Vlada0612 Following your advice the java dock was added.